dev/breeze/SELECTIVE_CHECKS.md

```diff
@@ -74,7 +74,7 @@ together using `pytest-xdist` (pytest-xdist distributes the tests among parallel
 ## Selective check decision rules
 
 * `Full tests` case is enabled when the event is PUSH, or SCHEDULE or we miss commit info or any of the
-  important environment files (setup.py, setup.cfg, provider.yaml, Dockerfile, build scripts) changed or
+  important environment files (pyproject.toml, Dockerfile, build scripts) changed or
```

Comment:
Same as above

Reply:
Not really. When provider.yaml file change, this might for example mean that only description of provider changed, or for example new Hook class have been added and we do not need to run `full tests` (or upgrade-to-newer-dependencies`. It's much better in this case to check if `generated/provider_dependencies.json` changed and `pyprroject.toml` changed (both will be updated by pre-commit. I added a note about it.